.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.4'
+services:
+  saas_app:
+    image: saas
+    build:
+      context: ../saas_app
+      dockerfile: ./Dockerfile
+    ports:
+      - 8000:8000
+    volumes:
+      - .:/workspace:cached
+    depends_on:
+      - postgresdb
+  postgresdb:
+    image: postgres
+    volumes:
+      - ../pgdata:/var/lib/postgresql/data
```

Comment:
`../pgdata` creates a local `pgdata` directory. We should either just leave the `/var/lib/postgresql/data` or include `/pgdata` in `.gitignore`.